backend/app/services/dashboard.py

```
@@ -160,17 +162,57 @@ def _fetch(ticker: str) -> tuple[str, float | None, bool]:
+# Maps the derived ``position.strategy`` lifecycle label to the V0.5 wheel
+# status pill (spec §14.6). Unknown values collapse to "Holding" so a
+# pre-recompute import doesn't crash the dashboard.
+_STRATEGY_TO_WHEEL_STATUS: dict[str, str] = {
+    "csp": "CSP",
+    "cc": "CC",
+    "wheel": "Wheel",
+    "holding": "Holding",
+}
+
+
+def _derive_wheel_status(strategy: str, has_open_call: bool, has_open_put: bool) -> str:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)